src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
Hey @damnko, it's totally fine! With the first comment, the thing is that at the moment looks like you are calling non-exising onClick(..) method, when you need to call onClick property by doing like you did here https://github.com/akveo/ng2-smart-table/pull/144/files#diff-a56dc4441e739f28ca9a61ac3b73caddR14

For the second comment - the AOT mode has much more stricter rules, thus we just asking you to make sure the code is building even with AOT in the first place.